### ISSUE TITLE
Page refresehes when trying to send a message

### ISSUE THREAD

**Issue by justus-saul** [lianglee/OhYesChat#21]
Hi,
when I try to send a message the page starts to refresh instead of sending the message.
OhYesChat Version: 2.2 (github master)
Elgg Version: 1.8.19
Browser: Chrome 34

Maybe the screenshot helps.
![screenshot at 13-44-53](https://cloud.githubusercontent.com/assets/5861826/2970567/3a044400-db5d-11e3-87c0-841fd8a57c6a.png)

**Comment by arsalanshah** [comment]
Elgg 1.8.19 is shipped with jquery 1.6.x we didn't tested it with jquery 1.11.x

**Comment by justus-saul** [comment]
Ok, thanks for the hint with jquery. As I never changed it myself one of the installed plugins must have done this. Will have a look at this. Btw. I tried the last 1.X version of OhYesChat and I was able to sent message to other user, but sometimes they did not arrive. But no refreshing as in 2.X.

**Comment by justus-saul** [comment]
Yes, it was related to my theme, modified version of: https://github.com/hypeJunction/hypeThemeX
It updates jquery. When deactivating it, everything was working. But as the Theme is more important than the chat, this won't help me, but at least I know the reason ;)

**Comment by upasamy** [comment]
i have same problem for my elgg 1.9.1,( Page refresehes when trying to send a message ), 

but when page in .../chat/messages/admin  is opened, the problem is fix.

how to fix it?

**Comment by lianglee** [comment]
we will look the issue and fix as soon as possible, at the moment we are working on http://opensource-socialnetwork.org

**Comment by upasamy** [comment]
ok thanks for your answer. 

I have opened the project you are working on, and I also have tried the demo. 

Whay the design imitate of facebook design?

On Tuesday, September 16, 2014 12:39 AM, Arsalan Shah notifications@github.com wrote:

we will look the issue and fix as soon as possible, at the moment we are working on opensource-socialnetwork.org

**Comment by lianglee** [comment]
@upasamy we are developing cms similar to facebook, but soon we will change layout.
